src/Dialect/ONNX/IndexExpr.hpp

```diff
@@ -285,6 +285,12 @@ class IndexExprContext {
   void createStoreOp(
       Value val, Value memref, SmallVectorImpl<IndexExpr> &indices);
 
+  // Emit a krnl.load to load an element from the memref.
+  Value createKrnlLoadOp(Value memref, SmallVectorImpl<IndexExpr> &indices);
```

Comment:
Is there still a need for the regular "createLoadOp" and "createStoreOp"? I would think that they can be removed

Reply:
I removed them. Thanks!